node-fs-readfile/cat.js

```diff
@@ -0,0 +1,17 @@
+import { readFile } from 'node:fs/promises';
+
+const args = process.argv.slice(2);
+
+async function cat() {
+  try {
+    for (let i = 0; i < args.length; i++) {
+      const text = args[i];
+      const contents = await readFile(`./${text}`, 'utf8');
+      console.log(contents);
+    }
```

Comment:
A for-loop works correctly, but if you use Promise.all it is more efficient since all the file reading happens at the same time.